lib/util/player_configuration.js

```diff
@@ -101,6 +101,7 @@ shaka.util.PlayerConfiguration = class {
         ignoreDrmInfo: false,
         xlinkFailGracefully: false,
         defaultPresentationDelay: 10,
+        ignoreMinBufferTime: false,
```

Comment:
This should also be added to externs/shaka/player.js, in the `shaka.extern.DashManifestConfiguration` type.  Please add `@property` to describe the property, and also add it to the `@typedef` block.

This will probably produce a few build errors, as the DASH config object in several of our tests will need to be updated to match the updated type.  Be sure to run `python build/all.py` and fix any errors that come up.

After that, please add a new test in `test/dash/dash_parser_manifest_unit.js` that shows the effect of ignoreMinBufferTime.  Thanks!

Reply:
i'm not yet sure on how to write a test properly but will check it later on

Reply:
I would suggest looking at the very last test in that file ('exposes Representation IDs') as a template.

The important parts of the test are:

1. Fake manifest text.  This can be very simple for your case, with only one Representation.
2. A call to `fakeNetEngine.setResponseText()` so the fake manifest text gets returned from the networking engine.
3. Any necessary configuration passed to `parser.configure()`.  For you, this should set the new flag.
4. The (async) call to `parser.start()` to parse the manifest and get the results.
5. Expectations on the output.  You would expect that minBufferTime is 0, in spite of the attribute in the manifest text.

Does that help?

Reply:
definitly help, thanks